Integrations/integration-fireeye.yml

```diff
@@ -468,3 +516,60 @@ script:
       description: FireEye application ID
     - contextPath: FFireEyeAX.Sensors.Profiles.Applications.-name
       description: FireEye application name
+    description: commands.server.feConfig.description
+  - name: fe-submit-url
+    arguments:
+    - name: analysistype
+      required: true
+      description: 'Specify live or sandbox analysis mode. 0—Live, analyze suspected
+        malware objects live within the MAS Multivector Virtual Execution (MVX) analysis
+        engine. 1—Sandbox, analyze suspected malware objects in a closed, protected
+        environment. Example: analysisType=live'
+    - name: profiles
+      required: true
+      description: Select the MAS profile to use for analysis. To determine the available
+        profiles, use the configuration command
+    - name: application
+      required: true
+      description: 'Specifies the application to be used for the analysis. To determine
+        the available applications  for a specific profile, use the configuration
+        command. Note: Setting the application value to 0, allows the MAS to determine
+        the file type for you'
+    - name: priority
+      required: true
+      description: 'Sets the analysis priority: 0—Normal, adds analysis to the bottom
+        of queue. 1—Urgent, places the analysis at the top of the queue'
+    - name: force
+      required: true
+      description: 'Specify whether to perform an analysis on the malware object even
+        if the object exactly matches an analysis that has already been performed.
+        In most cases, it is not necessary to reanalyze malware. (default: false)
+        0—False, Do not analyze duplicate objects. 1—True, Force analysis'
+    - name: prefetch
+      required: true
+      description: Specifies whether to determine the file target based on an internal
+        determination rather than browsing to the target location. 0—No 1—Yes
+    - name: timeout
+      required: true
+      description: Sets the analysis timeout (in seconds)
+    - name: urls
+      required: true
+      description: url to be analyzed
```

Comment:
Please write and add an example of URLs so it would be clear that it should be comma separated. Such as: `For example, "urls=www.demisto.com,www.google.com"`

Reply:
Included example for clarification